modules/3d-tiles/src/lib/parsers/parse-3d-tile-batched-model.js

```diff
@@ -12,10 +12,16 @@ import {parse3DTileGLTFViewSync, extractGLTF, GLTF_FORMAT} from './helpers/parse
 export async function parseBatchedModel3DTile(tile, arrayBuffer, byteOffset, options, context) {
   byteOffset = parseBatchedModel(tile, arrayBuffer, byteOffset, options, context);
   await extractGLTF(tile, GLTF_FORMAT.EMBEDDED, options, context);
+
+  const {extensions} = tile.gltf;
```

Comment:
Should this go into the function below instead, next to the `featureTable.getGlobalProperty('RTC_CENTER', GL.FLOAT, 3);`?

Reply:
The function below is synchronous. This field can only be accessed after the GLTF is loaded.